src/plugins/numberline/numberline-tile.tsx

```diff
@@ -1,14 +1,93 @@
+import React, { useEffect, useRef, useState } from "react";
 import { ITileProps } from "../../components/tiles/tile-component";
+import { scaleLinear, select, selectAll, pointer, axisBottom } from "d3";
 
 import "./numberline-tile.scss";
 
 export const NumberlineToolComponent: React.FC<ITileProps> = observer((props) => {
+  //---------------- Create unique className for tile ------
+  const tileId = props.model.id;
+  const axisClass = "axis-" + tileId;
+  const tileTitle = props.model.title;
+  //---------------- Calculate width of tile ---------------
+  const documentScrollerRef = useRef<HTMLDivElement>(null);
+  const [tileWidth, setTileWidth] = useState(0);
+  const containerWidth = (tileWidth * 0.93);
+  const axisWidth = (tileWidth * 0.9);
+  //pixels we shift to the right to center axis in numberline-tool-container
+  const xShiftRaw = ((containerWidth - axisWidth)/2);
+  const numToPx = (num: number) => num.toFixed(2).toString() + "px";
+
+  useEffect(() => {
+    let obs: ResizeObserver;
+    if (documentScrollerRef.current) {
+      obs = new ResizeObserver(() => {
+        setTileWidth(documentScrollerRef.current?.clientWidth ?? 0);
+      });
+      obs.observe(documentScrollerRef.current);
+    }
+    return () => obs?.disconnect();
+  }, []);
+  //----------------- Global Max Min -----------------------
+  const domainMin = -5;
+  const domainMax = 5;
+  const numOfTicks = domainMax - domainMin;
+
+  //----------------- Create Numberline Axis  --------------
+  useEffect(()=>{
+    // Construct axis
+    const linearScale = scaleLinear()
+    .domain([domainMin, domainMax])
+    .range([0, axisWidth]);
+    const axis = axisBottom(linearScale).tickSizeOuter(0);
+    axis.ticks(numOfTicks);
+    (select(`.${axisClass}`) as d3.Selection<SVGSVGElement, unknown, HTMLElement, unknown>).call(axis);
+
+    // After the axis is drawn, customize "x = 0 tick"
+    selectAll("g.num-axis g.tick line")
+      .attr("y2", function(x){ return (x === 0) ? 20 : 6;})
+      .attr("stroke-width", function(x){ return (x === 0) ? "3px" : "1px";})
+      .attr("style", function(x){ return (x === 0) ? "transform: translateY(-10px)" : "";});
+
+    //Set click-area to printNumber out
+    const printNum = (e: Event) => {
+      const pos = pointer(e, clickArea);
+      const xPos = pos[0];
+      const value = linearScale.invert(xPos);
+      //right now value is not being used but will need to be used eventually
+    };
+
+    select('.click-area')
+      .attr('width', axisWidth)
+      .on('click', (e) => printNum(e));
+
+    const clickArea = select('.click-area').node();
+
+  },[axisClass, axisWidth, domainMin, domainMax, numOfTicks]);
+
   return (
-    <div className="numberline-tool">
-      <textarea />
+    <div className="numberline-tool" ref={documentScrollerRef}>
+      <div className="numberline-tool-container">
+        <div className="num-axis-title-container">
+          <div className="title-box">
+            {tileTitle}
```

Comment:
In my current branch/next ticket (numberline plot points ) - I've made a wrapper (see below) 
<img width="1079" alt="image" src="https://github.com/concord-consortium/collaborative-learning/assets/521934/346223e9-ee29-4173-9946-6a37374d22d2">

Which is using our <BasicEditableTileTitle> component.